### ISSUE TITLE
Loglevel Settings

### ISSUE THREAD

**Issue by flunderveiko** [xsteadfastx/jitsiexporter#6]
Currently the log level is set to `info`, could a loglevel parameter be introduced?

**Comment by xsteadfastx** [comment]
do you use the docker image? the exporter itself has an debug switch. its just enabled in the Dockerfile (https://github.com/xsteadfastx/jitsiexporter/blob/master/Dockerfile#L15). maybe you should build your own docker image with the debug switch on false.

**Comment by flunderveiko** [comment]
Yes, I use the Docker image and also the parameter --debug = false, but I would like to be able to set the loglevel to info or warning if this is possible.
